specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -5057,6 +5057,180 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/networkConfig/virtualNetwork": {
```

Comment:
Is networkConfig a singleton? not sure if virtualNetwork is a resourceType or an instance of networkConfig? To close on this quicker, please feel free to set up a review with armapireview.